src/arrayMethodPop.js

```diff
@@ -5,7 +5,15 @@
  */
 function applyCustomPop() {
   [].__proto__.pop2 = function() {
-    // write code here
+    const result = this[this.length - 1];
+
+    if (this.length === 0) {
```

Comment:
якщо ти вже робиш перевірку, то роби її спочатку, щоб вийти з програми і нічого дальше не робити
```suggestion
    if (!this.length) {
```